lib/transactions.js

```diff
@@ -21,10 +21,21 @@ exports.txHash = () => txHash
 
 exports.createGeneration = (rpcData, blockReward, feeReward, recipients, poolAddress, poolHex, coin, masternodeReward, masternodePayee, masternodePayment) => {
     let poolAddrHash = bitcoin.address.fromBase58Check(poolAddress).hash
-    let tx = new bitcoin.Transaction()
 
-    if (coin.overwinter) {
-        tx.setOverwinter()
+    let network = bitcoin.networks[coin.symbol]
```

Comment:
This might cause issues with testnet coins not having the same symbol as mainnet coins if they have custom params in `bitgo-utxo-lib`. Just mentioning it so we keep it in mind in the future.